contracts/Cartographer.sol

```diff
@@ -333,6 +336,35 @@ contract Cartographer is Ownable, Initializable, ReentrancyGuard {
     // --   M O D I F I E R S (Many are split to save contract size)
     // -----------------------------------------------------------------
 
+    function _getTaxBP(address _userAdd, address _token)
```

Comment:
Can you move this and _bonusBP to the top of the `POOL INTERACTIONS` section so it is closer to where it is used.